lobby_server.yaml

```diff
@@ -2,6 +2,9 @@
   host: lobby.triplea-game.org
   port: 3304
   message: |
+    IMPORTANT: The Lobby will be down for some maintenance on Thursday, the 30th November 2017 at 9:00am UTC.
+    Sorry for your inconvenience.
```

Comment:
If possible, we should provide a window during which the lobby will be down so users have an idea when they can play online again.  Plus, that will time box the upgrade in case we run into any problems (i.e. start rollback procedures if there's only X minutes left in the window).

Actually, a nice feature for the future would be the ability to run a lobby server in "maintenance mode."  All it would do is fail all authentication requests with an error message that indicates the lobby is down for maintenance.  Then you could just run any version of the lobby with that feature while you do your maintenance.  When everything is complete, stop the dummy lobby server, and start the real (possibly upgraded) version.

Reply:
Awesome idea ^^

Reply:
@ssoloff Goo Idea, could be error prone when it comes to port binding though...